agentuity/instrument/__init__.py

```diff
@@ -116,8 +116,8 @@ def configure_native_provider(agentuity_url: str, agentuity_api_key: str) -> boo
 
 
 def instrument():
-    agentuity_url = os.getenv("AGENTUITY_URL", None)
-    agentuity_api_key = os.getenv("AGENTUITY_API_KEY", None)
+    agentuity_url = os.getenv("AGENTUITY_TRANSPORT_URL", "https://agentuity.ai")
```

Comment:
@robindiddams should we keep this AGENTUITY_URL? otherwise we will need to change CLI and Nucleus too